pkg/hubble/parser/options/options.go

```diff
@@ -61,6 +62,18 @@ func Redact(logger *slog.Logger, httpQuery, httpUserInfo, kafkaApiKey bool, allo
 	}
 }
 
+// EnableL3L4PolicyEnrichment configuInfo("configured Hubble with L3/L4 network policy enrichment")res the Network Policy enrichment of Hubble Flows.
+func EnableL3L4PolicyEnrichment(logger *slog.Logger, enableL3L4 bool) Option {
```

Comment:
nitpicking: can we prefix this func name With for consistency? I know `Redact()` doesn't follow this convention, but many other options do.

Reply:
Done. I can update Redact also to follow this convention if needed.